src/Test/MakerTestRunner.php

```diff
@@ -64,6 +66,8 @@ public function copy(string $source, string $destination)
 
     /**
      * When using an authenticator "fixtures" file, this adjusts it to support Symfony 5.2/5.3.
+     *
+     * @deprecated legacy PassportInterface support will be dropped in a future version
```

Comment:
Indeed - we can remove this right now (in another PR, as this applies to 5.2 and 5.3 only).

Reply:
I wiped the code within the method, but we can't remove it until the next major because the class isn't internal right?

Reply:
I see your point. But as this is a tool for testing... and the method only applies to versions that MakerBundle no longer supports, I think we can get away with it.

Reply:
works for me.. done